_maps/yogstation/RandomRuins/StationRuins/BoxStation/bar_purple.dmm

```diff
@@ -198,6 +198,7 @@
 	icon_state = "tile_corner";
 	dir = 8
 	},
+/obj/item/storage/box/goatcubes,
```

Comment:
just make this spawn in `/obj/structure/closet/secure_closet/freezer/meat` instead of putting it on every map :)